modules/io/common/src/main/java/org/locationtech/jts/io/geojson/FeatureCollection.java

```diff
@@ -0,0 +1,24 @@
+package org.locationtech.jts.io.geojson;
+
+import java.util.ArrayList;
+
+/**
+ * A FeatureCollection in GeoJSON contain a collection of zero or more features.
+ * <p>
+ * A specification of the GeoJson format can be found at the GeoJson web site:
+ * <a href='http://geojson.org/geojson-spec.html'>http://geojson.org/geojson-spec.html</a>.
+ * <p>
+ */
+public class FeatureCollection extends ArrayList<Feature> {
```

Comment:
Not sure its good to extend ArrayList.   It introduces a big dependency which might be brittle over time.

Reply:
Perhaps just let the API accept a raw `List<Feature>` (or even `Collection<Feature>`) for writing, and return `List<Feature>` for reading.